server/terraform/README.md

```diff
@@ -222,19 +222,33 @@ Analytics may be disabled by the user on their device.
    - Prod: use WHO existing analytics account
    - For both, click "Enable Google Analytics"
 1. Click on link below "Linked Google Analytics account"
+
+#### Analytics Account Settings
+
+1. Visit Analytics Page (same as above link):
+   - Dev: https://analytics.google.com/analytics/web/?authuser=0#/a183548134p253618398/admin
+   - Prod: ????
 1. Select "Account Settings"
-1. Configure "Country of Business" (doesn't change where data is processed)
-   - Dev: United States
-   - Prod: Switzerland
-1. Click "Save" at bottom of the page
+   1. Configure "Country of Business" (doesn't change where data is processed)
+      - Dev: United States
+      - Prod: Switzerland
+   1. For "Data Sharing Settings", disable "Google products & services"
```

Comment:
Are you intentionally leaving the others enabled?
![100518749-65c8d900-3148-11eb-97d5-2318d2620f99](https://user-images.githubusercontent.com/11729521/100942773-c6da0f00-34b0-11eb-8265-e2dc4b513bac.jpeg)

Reply:
Yes. I want to get the feedback from WHO as well but here's my current thinking:
- Benchmarking: policy aggregates and anonymizes the data. Performance benchmarking is helpful and shouldn't have any sensitive data, certainly not in aggregate.
- Technical support / Account specialists: I expect that WHO will need assistance in various matters. I don't want them to be stuck with a large problem for which they can't seek advice from the experts. The Google privacy policy mentions an obligation of confidentiality and minimal access.